src/test/java/de/retest/recheck/persistence/bin/KryoPersistenceTest.java

```diff
@@ -92,4 +93,26 @@ void on_error_file_should_be_deleted() throws IOException {
 				.isInstanceOf( KryoException.class );
 		assertThat( nonexistent ).doesNotExist();
 	}
+
+	@Test
+	void isCompatible_should_be_compatible_with_equal_or_higher_value() throws Exception {
```

Comment:
Maybe merge these tests into a parameterized one?

Reply:
That would be two versions, I don't think that is really worth.